### DISCUSSION TITLE
Spread emotion/react version import

### DISCUSSION THREAD

**Discussion by felipe-silva-devtech** [system-ui/theme-ui#2367] [General] [answered]
**Is your feature request related to a problem? Please describe.**
I'm using rollup to build my application and I'm getting some warning because rollup understands as a global module as can see in the follow image:

![image](https://user-images.githubusercontent.com/26209056/202544562-b72bc1bc-86db-4459-a5ef-7a20f631c653.png)

**Describe the solution you'd like**
My suggestion is spread the version on import, something like:

```
import { version as emotionReactVersion } from '@emotion/react/package.json'

...

const __EMOTION_VERSION__ = emotionReactVersion  
```

**Additional context**
References:

https://github.com/system-ui/theme-ui/blob/f5ae7d69bb3f63dff996f6ddf4705009d3b4429a/packages/core/src/index.ts#L8

https://github.com/system-ui/theme-ui/blob/f5ae7d69bb3f63dff996f6ddf4705009d3b4429a/packages/core/src/index.ts#L38

**Comment by hasparus** [comment, marked as answer]
Hey @felipe-silva-devtech, to be frank I don't think that your suggestion fixes your problem.

It may change the warning text, because `packageInfo` was inferred from our import, but it doesn't solve anything.

You are building an [UMD](https://github.com/umdjs/umd) build with Rollup, and it expects the package.json from Emotion to be assigned to a global variable. It doesn't know that variable, because you did not set `output.globals`, and it guesses `packageInfo`.

Thing is, I don't think there is such a global variable. Moreover, Theme UI doesn't build to UMD, so there won't be a global variable for it too. Are you confident that you need an UMD build? ESM and CJS builds are usually enough for the most common cases.
